src/_posts/platform/app/2000-01-01-notification.md

```diff
@@ -48,6 +48,9 @@ Note that Mattermost notifications require using the Slack notifier.
 * collaborator_accepted: Collaborator accepted, A collaborator invitation was accepted
 * collaborator_invited: Collaborator invited, A collaborator was invited
 * collaborator_removed: Collaborator removed, A collaborator was removed
+* database_maintenance_planned: Database Maintenance Planned, A database maintenance has been planned
+* database_maintenance_started: Database Maintenance Started, A database maintenance has started
+* database_maintenance_ended: Database Maintenance Ended, A database maintenance has ended
```

Comment:
Not sure if the capital letters are needed here as it is not the case for others.
```suggestion
* database_maintenance_planned: Database maintenance planned, A database maintenance has been planned
* database_maintenance_started: Database maintenance started, A database maintenance has started
* database_maintenance_completed: Database maintenance completed, A database maintenance has completed
```

Reply:
You are right, fixing